config/database.yml

```diff
@@ -51,8 +58,9 @@ development:
 # Do not set this db to the same as development or production.
 test:
   <<: *default
-  pool: 5
   database: recipe_app_test
+  username: postgres
+  password: $@Mr3b01
```

Comment:
Hi @Lancelot-SO  please remove your postgres user name and password. This will create database connection errors in other user's systems.